### ISSUE TITLE
Gmil brute

### ISSUE THREAD

**Issue by DomKing13** [Anontemitayo/Gmail-brute#1]
how do I ger to the passwors brute attack ? I dont know answer after 2 question yet? Can you lmk God bless.

**Comment by Anontemitayo** [comment]
Hello brother, Please kindly drop a screenshot of where you are stuck to assist me in knowing where to help

**Comment by Anontemitayo** [comment]
> 

Hello brother, Please kindly drop a screenshot of where you are stuck to assist me in knowing where to help

**Comment by DomKing13** [comment]
thank you for quick reply brother it is the second question asked after
typing in address

**Comment by Anontemitayo** [comment]
I guess you meant password list 
You can create a victim based password list manually or you can use my tool to create one 
:

apt update

apt upgrade

apt install python

apt install git

git clone https://github.com/Anontemitayo/Passmaker

cd Passmaker

Run :

python Passmaker.py

**Comment by DomKing13** [comment]
yes thank you still a newbe at cracking psswrds so I just write all the
possible words? So I thought there was a preset list or it just keeps
continuously guessing until it gets it have you ever heard of a program
that does that our do you know of any because I only have my phone at this
time to use.  Thanks for the help

**Comment by Anontemitayo** [comment]
No problem 
Do try to share my repo and tools with your friends 
And incase u wish to contact me 
facebook.com/Anontemitayo

**Comment by DomKing13** [comment]
one more ? and yes I will get the word asap of your directory or program I
mean. Does hashcat work in Gmail-Hack and can you give me an example if it
does bro?
